.github/workflows/rust-av.yml

```diff
@@ -1,101 +1,109 @@
 name: rust-av
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+    paths:
```

Comment:
This seems like a bit much. How about we just ignore `*.md`? We don't need to whitelist every filetype in the repo.